### ISSUE TITLE
Examples?

### ISSUE THREAD

**Issue by echo66** [PonyGE/PonyGE2#93]
Greetings! 

I'm looking for code examples on how to use PonyGE2 but I'm out of luck. Where should I look to learn more? Thanks in advance!

**Comment by dvpfagan** [comment]
The docs are fairly detailed and show how to run the code

Dave

**Comment by echo66** [comment]
Hello Dave! 

Does it show how to use the library classes and modules? Or just the command line?

**Comment by dvpfagan** [comment]
Shows how to run command line and also how to edit the parameters files to
for custom setups

What exactly are you trying to do?

Dave

**Comment by dvpfagan** [comment]
Nope doesn’t work that way yet. We set parameters and just run it in a
traditional way.

Dave

**Comment by echo66** [comment]
Thank you for the clarification.

**Comment by buddha314** [comment]
I agree I would like to see a "quick start" tutorial.  [This page](https://github.com/PonyGE/PonyGE2/wiki/Example-Problems) is basically "run this command line".  For me, fitting a polynomial would be extremely useful.

**Comment by dvpfagan** [comment]
@buddha314 so for the examples given in the quick start, the parameter files load the required parameters to do the run. For fitting a polynomial you just have to provide your samples into the train parameter as a txt file like Vladislavleva4/Train.txt and then select a grammar such as supervised_learning/Vladislavleva4.bnf then run the command. There has been a concerted effort from the development team to only provide realistic benchmarks problems in the software, as the EC field is trying to move towards using better benchmark problems as discussed here (https://cs.gmu.edu/~sean/papers/gecco12benchmarks3.pdf) and subsequent papers.
